### PR TITLE
Add fastify-url-data to ecosystem list

### PR DESCRIPTION
**Pull request by jsumners**
As the title says.

### CODE REVIEW

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Plugins maintainted by the fastify team are listed under [Core](#core) while plu
 - [`fastify-register-timeout`](https://github.com/fastify/fastify-register-timeout) Register plugin with a timeout.
 - [`fastify-swagger`](https://github.com/fastify/fastify-swagger) Swagger documentation generator for Fastify.
 - [`fastify-websocket`](https://github.com/fastify/fastify-websocket) WebSocket support for Fastify. Built upon [websocket-stream](https://github.com/maxogden/websocket-stream).
+- [`fastify-url-data`](https://github.com/fastify/fastify-url-data) Decorate the `Request` object with a method to access raw URL components.
```
Comment (StarpTech) [inline docs/Ecosystem.md]: Perhaps link to https://github.com/fastify/fastify/blob/master/docs/Request.md ?

**Comment by StarpTech** [comment]
LGTM

File: docs/Ecosystem.md (hunk 1 above)
Comment (jsumners) [reply docs/Ecosystem.md]: I think that would be difficult to maintain during documentation refactors. It should be safe to assume that someone looking for this plugin knows what the `Request` object is. But I'll change it if other @fastify/fastify members agree it should be linked.

Comment (mcollina) [reply docs/Ecosystem.md]: I don't think so, we are not linking swagger, ejs, pug, handlebars, marko, etc..

**Pull request merged**